redaxo/src/addons/media_manager/lib/media_manager.php

```diff
@@ -252,9 +252,9 @@ public function sendMedia()
                 unset($header['Last-Modified']);
             }
             foreach ($header as $t => $c) {
-                header($t . ': ' . $c);
+                rex_response::setHeader($t, $c);
             }
-            readfile($CacheFilename);
+            rex_response::sendFile($CacheFilename, $header['Content-Type']);
```

Comment:
gibts den header immer im array?

Reply:
wenn ich das richtig sehe, kann dies nur passieren, wenn kein das medium kein gültigen mime-type hat.

Reply:
weil ansonsten wenn die Datei in den Cache geschrieben wird, vorher der Content-Type versucht wird zu ermitteln.
https://github.com/redaxo/redaxo/blob/2b709201dd39ed9073be63122b1368e0557ee3c6/redaxo/src/addons/media_manager/lib/managed_media.php#L356-L379